scripts/sdf-glyph-foundry/0.1.1/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: generic
+sudo: false
```

Comment:
Done.... although I have very little understanding of what the change actually means.